rules_building_block/command_and_control_non_standard_http_port.toml

```diff
@@ -0,0 +1,89 @@
+[metadata]
+creation_date = "2023/07/10"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/07/10"
+
+[rule]
+author = ["Elastic"]
+building_block_type = "default"
+description = """
+Identifies potentially malicious processes communicating via a port paring typically not associated with HTTP/HTTPS.
+For example, HTTP over port 8443 or port 440 as opposed to the traditional port 80 , 443.
+Adversaries may make changes to the standard port a protocol uses to bypass filtering or
+muddle analysis/parsing of network data.
+"""
+from = "now-119m"
+interval = "60m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Non-Standard Port HTTP/HTTPS connection"
+risk_score = 21
+rule_id = "62b68eb2-1e47-4da7-85b6-8f478db5b272"
+severity = "low"
+tags = ["Domain: Endpoint", "OS: Linux", "OS: macOS", "Use Case: Threat Detection", "Tactic: Command and Control", "Rule Type: BBR"]
```

Comment:
https://github.com/elastic/detection-rules/pull/2912/files#r1259109521

Reply:
Responded by Jon on original PR referenced above

Reply:
Merging this with the same tag formats as the original referenced PR has the same tag formats.